src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback, useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorLoading, setErrorLoading] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const fetchpeople = useCallback(async () => {
+    setIsLoading(true);
+    setErrorLoading(false);
+    try {
+      // fetching data
+      const fetchedData = await getPeople();
+      // setting data to people
+
+      setPeople(fetchedData);
+    } catch (error) {
+      // error
```

Comment:
remove comments

your code should describe itself

comments are usually used for explaining WHY you did it in this way, not WHAT it does